### PR TITLE
upgrade global hub to release 2.9

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.9.yaml [hunk 1]
```diff
@@ -155,34 +155,34 @@ tests:
...
-- as: release-280-operator-image-mirror
+- as: release-290-operator-image-mirror
   postsubmit: true
   steps:
     dependencies:
       SOURCE_IMAGE_REF: multicluster-global-hub-operator
     env:
       IMAGE_REPO: multicluster-global-hub-operator
-      IMAGE_TAG: v0.7.0
+      IMAGE_TAG: v0.8.0
```
Comment (clyang82) [inline ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.9.yaml:165]: use 1.0.0 instead as we discussed this morning.

**Comment by clyang82** [comment]
https://github.com/openshift/release/blob/b8ccc8173257f54a17184e51428844c9dfab8071/ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main.yaml#L168
need to use `release-2.9`

**Comment by clyang82** [comment]
https://github.com/openshift/release/blob/b8ccc8173257f54a17184e51428844c9dfab8071/ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-main.yaml#L39 to `2.9`

**Review by clyang82 — CHANGES_REQUESTED** [review]

File: ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.9.yaml (hunk 1 above)
Comment (yanmxa) [reply ci-operator/config/stolostron/multicluster-global-hub/stolostron-multicluster-global-hub-release-2.9.yaml]: Done!

**Pull request merged**